### ISSUE TITLE
can't turn off debug on production

### ISSUE THREAD

**Issue by fgregg** [datamade/chi-councilmatic#286]
and were are doing a little leaking of implementation data: https://chicago.councilmatic.org/legislation/o2022-3464/

however, if we set debug to false in the heroku settings, images don't load and things are styled quite weirdly.

![Screenshot 2022-10-26 at 15-59-32 Chicago Councilmatic](https://user-images.githubusercontent.com/536941/198125059-1cbac141-4c42-4006-b3a7-14440d98d412.png)

**Comment by derekeder** [comment]
this is partially addressed in #287, but we will need to do #284 to fully resolve it.

After bringing in our [documented whitenoise changes for Heroku/s3 static files](https://github.com/datamade/how-to/blob/main/deployment/heroku/deploy-a-django-app.md#serve-static-files-with-whitenoise), I'm getting an error about a missing static leaflet resource that lives in `django-councilmatic`:

```
Post-processing 'css/leaflet.css' failed!

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 413, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 354, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.7/site-packages/django/core/management/base.py", line 398, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 187, in handle
    collected = self.collect()
  File "/usr/local/lib/python3.7/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 134, in collect
    raise processed
whitenoise.storage.MissingFileError: The file 'css/images/layers.png' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0xffff74b72390>.

The CSS file 'css/leaflet.css' references a file which could not be found:
  css/images/layers.png

Please check the URL references in this CSS file, particularly any
relative paths which might be pointing to the wrong location.
```

**Comment by derekeder** [comment]
ok actually, my latest in #288 allows debug=False to work. We'll still want to sort out the missing static file issue in #283 and #284
